pulsar/managers/util/drmaa/__init__.py

```diff
@@ -41,13 +41,14 @@ class DrmaaSession(object):
 
     def __init__(self, session_constructor, **kwds):
         with self._session_lock():
+            DrmaaSession.session_count += 1
```

Comment:
Heh... but now it will trigger the warning on line 47. This should come after the if block starting on line 45.